.github/workflows/test-lint.yml

```diff
@@ -22,10 +22,11 @@ jobs:
         with:
           PATTERNS: |
             **/*.go
-            */go.mod
-            */go.sum
+            **/go.mod
+            **/go.sum
```

Comment:
Just a comment regarding this change, the intention of using a single "*" was to enforce that all plugin packages are defined in root folders to avoid nesting. That said, does it make sense to track internal packages, a part from the "root defined" ones?